modules/ROOT/pages/monitor-applications-on-rtf.adoc

```diff
@@ -6,10 +6,9 @@ Monitoring for applications deployed to Runtime Fabric is enabled by default.
 
 Anypoint Monitoring supports the following Mule versions for Runtime Fabric deployments:
 
-* Mule Runtime 3.8.7-AM
-* Mule Runtime 3.9.1
-* Mule Runtime 3.9.2
-* Mule Runtime, version 4.1.2+
+* Mule Runtime 3.8.7
```

Comment:
GLOBALLY change these to either Mule or Mule runtime engine rather than Mule Runtime.